fix build errors when using Doxygen 1.8.16 like:
    OpenMMSwigHeaders.i:12: Error: Bad $descriptor() macro.
see https://github.com/openmm/openmm/issues/2420 and https://github.com/openmm/openmm/pull/2460
diff --git a/plugins/amoeba/wrappers/Doxyfile.in b/plugins/amoeba/wrappers/Doxyfile.in
index a35fd31b7..1fd04e509 100644
--- a/plugins/amoeba/wrappers/Doxyfile.in
+++ b/plugins/amoeba/wrappers/Doxyfile.in
@@ -581,7 +581,7 @@ INPUT_ENCODING         = UTF-8
 # *.c *.cc *.cxx *.cpp *.c++ *.java *.ii *.ixx *.ipp *.i++ *.inl *.h *.hh *.hxx
 # *.hpp *.h++ *.idl *.odl *.cs *.php *.php3 *.inc *.m *.mm *.py *.f90

-FILE_PATTERNS          = 
+FILE_PATTERNS          = *.h

 # The RECURSIVE tag can be used to turn specify whether or not subdirectories
 # should be searched for input files as well. Possible values are YES and NO.
diff --git a/wrappers/Doxyfile.in b/wrappers/Doxyfile.in
index 76f74cfc6..ce7b7a8c4 100644
--- a/wrappers/Doxyfile.in
+++ b/wrappers/Doxyfile.in
@@ -582,7 +582,7 @@ INPUT_ENCODING         = UTF-8
 # *.c *.cc *.cxx *.cpp *.c++ *.java *.ii *.ixx *.ipp *.i++ *.inl *.h *.hh *.hxx
 # *.hpp *.h++ *.idl *.odl *.cs *.php *.php3 *.inc *.m *.mm *.py *.f90

-FILE_PATTERNS          = 
+FILE_PATTERNS          = *.h

 # The RECURSIVE tag can be used to turn specify whether or not subdirectories
 # should be searched for input files as well. Possible values are YES and NO.
diff --git a/wrappers/python/src/swig_doxygen/doxygen/Doxyfile.in b/wrappers/python/src/swig_doxygen/doxygen/Doxyfile.in
index a1e664ae6..c835beeb9 100644
--- a/wrappers/python/src/swig_doxygen/doxygen/Doxyfile.in
+++ b/wrappers/python/src/swig_doxygen/doxygen/Doxyfile.in
@@ -588,7 +588,7 @@ INPUT_ENCODING         = UTF-8
 # *.c *.cc *.cxx *.cpp *.c++ *.java *.ii *.ixx *.ipp *.i++ *.inl *.h *.hh *.hxx
 # *.hpp *.h++ *.idl *.odl *.cs *.php *.php3 *.inc *.m *.mm *.py *.f90

-FILE_PATTERNS          = 
+FILE_PATTERNS          = *.h

 # The RECURSIVE tag can be used to turn specify whether or not subdirectories
 # should be searched for input files as well. Possible values are YES and NO.
